Index: babuluo/web/app/product/product/productAdd/controllers.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- babuluo/web/app/product/product/productAdd/controllers.js	(revision 69768db84625e1ee2d245b5c05e1afc99e71f50a)
+++ babuluo/web/app/product/product/productAdd/controllers.js	(revision )
@@ -88,6 +88,8 @@
 
     $scope.addProductSubmit = function () {
         $scope.add['shop_product.KEYWORD'] = getTabInputText($scope.keyword);
+        $scope.add['shop_product.CLASS_ID'] = $scope.add['shop_product_class.CLASS_ID'];
+        $scope.add['shop_product.UNIT_ID'] = $scope.add['shop_unit.UNIT_ID'];
         var form = $scope.add;
         form['shop_product.SERVICE_ID'] = 1;
         form['shop_product.PRD_SPU'] = $scope.spuCode;
Index: .idea/libraries/lib.xml
===================================================================
--- .idea/libraries/lib.xml	(revision 69768db84625e1ee2d245b5c05e1afc99e71f50a)
+++ .idea/libraries/lib.xml	(revision 69768db84625e1ee2d245b5c05e1afc99e71f50a)
@@ -1,13 +0,0 @@
-<component name="libraryTable">
-  <library name="lib">
-    <CLASSES>
-      <root url="file://$PROJECT_DIR$/babuluo/web/WEB-INF/lib" />
-    </CLASSES>
-    <JAVADOC />
-    <SOURCES>
-      <root url="file://$PROJECT_DIR$/babuluo/web/WEB-INF/lib" />
-    </SOURCES>
-    <jarDirectory url="file://$PROJECT_DIR$/babuluo/web/WEB-INF/lib" recursive="false" />
-    <jarDirectory url="file://$PROJECT_DIR$/babuluo/web/WEB-INF/lib" recursive="false" type="SOURCES" />
-  </library>
-</component>
\ No newline at end of file
Index: babuluo/web/app/product/product/productList/controllers.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- babuluo/web/app/product/product/productList/controllers.js	(revision 69768db84625e1ee2d245b5c05e1afc99e71f50a)
+++ babuluo/web/app/product/product/productList/controllers.js	(revision )
@@ -8,7 +8,6 @@
             modalFactory.showShortAlert("jaja");
         });
     };
-    //$scope.initLoad();
 
     $scope.bindData = function (response) {
         $scope.productSpuSizeMap = new Map;
@@ -19,6 +18,7 @@
         $scope.checkedSku = [];
         $scope.checkAllProduct = false;
 
+        console.log(response);
         filterTableFromList(response.data, "shop_product");
         $scope.productList = response.data;
 
